sources/osgShadow/ShadowMap.js

```diff
@@ -43,14 +43,18 @@ utils.createPrototypeObject(
             var near = nv.getComputedNear(),
                 far = nv.getComputedFar();
 
+            if (near === Infinity && far === -Infinity) {
```

Comment:
are you sure it's && and not a ||

Reply:
yes.
It checks for exact state  'no geom at all did pass the cullvisitor cull'.
So it check the exact state  CullVisitor resets near and far to before doing a cull. if either is not '===' it would mean something has passed the test and a geom is drawn.